index.html

```diff
@@ -15,64 +15,159 @@
 
       <!-- Navigation bar section -->
       <div class="row">
-        <div class="column">
+        <div class="col">
 
         </div>
       </div>
 
 
       <!-- Intro section -->
       <div class="row">
-        <div class="column">
+        <div class="col">
 
         </div>
       </div>
 
 
       <!-- What is section -->
       <div class="row">
-        <div class="column">
+        <div class="col">
 
         </div>
       </div>
 
 
       <!-- Atomist enables, cards section -->
       <div class="row">
-        <div class="column">
+        <div class="col">
 
         </div>
       </div>
 
 
       <!-- Powered by atomist section -->
       <div class="row">
-        <div class="column">
+        <div class="col">
 
         </div>
       </div>
 
 
       <!-- Latest from atomist section -->
       <div class="row">
-        <div class="column">
-
+        <div class="col">
+          <h3>The Latest from Atomist</h3>
+          <div class="card">
+            <img src="wordcloud.jpg" alt="">
+            <h4>Posted by Rod Johnson</h4>
+            <h3>Evolving Understanding of Delivery</h3>
+            <p>
+              Two new terms have recently emerged around software delivery: Software Defined 
+              Delivery and Progressive Delivery. Why? How do they relate to Continuous Delivery?
+              Several forces today make delivery increasingly complex. Notably, proliferation of 
+              repositories, with hundreds of small projects replacing a handful of monoliths; 
+              desire for greater automation to realize the full potential of CD across multiple 
+              environments; the rise of feature flagging; and increased evidence (such as the 
+              Equifax debacle)
+            </p>
+            <button>Read article</button>
+          </div>
+          <div class="card">
+            <img src="l8istsh9ny.jpg" alt="">
+            <h4>Posted by Ryan Day</h4>
+            <h3>Event-Driven Software Delivery</h3>
+            <p>
+              Atomist co-founder and Chief Architect, Jim Clark, recently sat in with Rob Hirschfeld 
+              for an episode of the L8ist Sh9y Podcast. It's a wide-ranging conversation about delivery 
+              as workflow, event-driven delivery, code analysis, GraphQL and ChatOps.
+            </p>
+            <button>Read article</button>
+          </div>
+          <div class="card">
+            <img src="Podcast-Makers-1.jpg" alt="">
```

Comment:
Try enclosing the content of the card in a card-body tag `<div class="card-body">` to keep consistency.